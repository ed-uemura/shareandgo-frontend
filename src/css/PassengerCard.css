.passenger-picture {
	display: flex;
	background-color: var(--SGGreen);
	width: 42px;
	height: 42px;
	border-radius: 100%;
	overflow: hidden;
	justify-content: center;
	margin: 0;
	left: 0;
}

.passenger-picture img {
	height: 100%;
}
