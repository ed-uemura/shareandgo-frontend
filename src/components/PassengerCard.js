import React from 'react';
import '../css/PassengerCard.css';

const PassengerCard = ({ image, name }) => {
	return (
		<>
			<figure className='passenger-picture'>
				<img src={image} alt={name} />
			</figure>
			<p className='ms-2'>{name}</p>
		</>
	);
};

export default PassengerCard;
