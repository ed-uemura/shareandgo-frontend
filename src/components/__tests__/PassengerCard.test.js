import {render, screen, cleanup} from '@testing-library/react'
import renderer from 'react-test-renderer'
import PassengerCard from '../PassengerCard';
import DriverImage from '../../images/profile-picture-mockup.jpg';

afterEach(() =>{
    cleanup();
});

test('matches snapshot', () =>{
    const name = 'Betty Wallace';
    const tree = renderer.create(<PassengerCard name={name} image={DriverImage}/>).toJSON();
    expect(tree).toMatchSnapshot();
})

