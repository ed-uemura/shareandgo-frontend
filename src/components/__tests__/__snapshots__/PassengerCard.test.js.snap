// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  <figure
    className="passenger-picture"
  >
    <img
      alt="Betty Wallace"
      src="profile-picture-mockup.jpg"
    />
  </figure>,
  <p
    className="ms-2"
  >
    Betty Wallace
  </p>,
]
`;
